index.js

```diff
@@ -126,6 +126,35 @@ cm.Item({
   }
 });
 
+cm.Item({
+  label: contextMenuLabel,
+  context: [
+    cm.URLContext(['*.google.com']),
+    //TODO?: create a variable with all supported hrefs and reference it in this SelectorContext
+    cm.SelectorContext('[href*="youtube.com%2F"], [href*="youtu.be%2F"], [href*="vine.co%2F"], [href*="vimeo.com%2F"]')
+  ],
+  contentScript: contextMenuContentScript,
+  onMessage: function(url) {
+    const regex = /url=(https[^;]*)/;
+    const match = regex.exec(url)[1];
+    const decoded = decodeURIComponent(match).split('&usg')[0];
+    let getUrlFn;
+    if (decoded.indexOf('youtube.com' || 'youtu.be') > -1) {
+        getUrlFn = getYouTubeUrl;
+        domain = 'youtube.com';
+      } else if (decoded.indexOf('vimeo.com')  > -1) {
+        getUrlFn = getVimeoUrl;
+        domain = 'vimeo.com';
+      } else if (decoded.indexOf('vine.co') > -1) {
+        getUrlFn = getVineUrl;
+        domain = 'vine.co';
+      }
+    launchVideo({url: decoded,
```

Comment:
A weird edgecase, but we should check if `domain` and `getUrlFn` are defined here before calling `launchVideo`